.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
I've never seen a "nested" .gitignore file. If all of .idea/ is moved to the main .gitignore file then /workspace.xml would also be ignored inherently. When pulling this branch down to local it also pulled the .idea/ directory even though it appears in .gitignore. Maybe there is some sort of conflict with the 2 .gitignore files?